diff --git a/CMakeLists.txt b/CMakeLists.txt
index ee18d69..4f5de01 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -75,14 +75,14 @@ target_link_libraries(taihen
 )
 
 vita_create_self(taihen.skprx taihen
-  CONFIG ${CMAKE_SOURCE_DIR}/exports.yml
+  CONFIG ${CMAKE_CURRENT_SOURCE_DIR}/exports.yml
   UNSAFE
 )
-vita_create_stubs(taihen-stubs taihen ${CMAKE_SOURCE_DIR}/exports.yml
+vita_create_stubs(taihen-stubs taihen ${CMAKE_CURRENT_SOURCE_DIR}/exports.yml
   KERNEL
 )
 
-install(DIRECTORY ${CMAKE_BINARY_DIR}/taihen-stubs/
+install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/taihen-stubs/
   DESTINATION lib
   FILES_MATCHING PATTERN "*.a"
 )

