--- vitasdk/share/vita.cmake	2020-01-09 14:10:23.000000000 -0500
+++ vitasdk2/share/vita.cmake	2020-01-09 21:22:57.699001000 -0500
@@ -84,12 +84,12 @@
   )

   ## SELF target
-  add_custom_target(${target}
+  add_custom_target(${target}.dep
     ALL
     DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/${target}
   )
   if(TARGET ${source})
-    add_dependencies(${target} ${source})
+    add_dependencies(${target}.dep ${source})
   endif()
 endmacro(vita_create_self)
 ##################################################
@@ -239,12 +239,12 @@
   )

   ## VPK target
-  add_custom_target(${target}
+  add_custom_target(${target}.dep
     ALL
     DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/${target}
   )
   if(TARGET ${eboot})
-    add_dependencies(${target} ${eboot})
+    add_dependencies(${target}.dep ${eboot})
   endif()
 endmacro(vita_create_vpk)
 ##################################################
